helpers/z_schema.js

```diff
@@ -41,10 +41,6 @@ z_schema.registerFormat('address', function (str) {
 });
 
 z_schema.registerFormat('username', function (str) {
-	if (str.length === 0) {
```

Comment:
@Isabello @karmacoma, we accept an empty string as username format (But we return an error if the username is empty somewhere else in the verification process), It did not make sense to me so i removed accepting empty strings. Let me know if we need to revert this thing.

Reply:
@SargeKhan We should revert that removal. Length is handled by other schema constraints.

Reply:
@SargeKhan We should revert this removal. Length is handled by other schema constraints.